docs/internal/config/Lookup-Tables.md

```diff
@@ -0,0 +1,53 @@
+## Viewing lookup tables
+
+Lookup tables contain data which may be referenced by application forms. 
+
+### To view the list of lookup tables: 
+To find the menu (visible only to Admin users), go to: Configurations > Lookup Tables
+![Loopup tables tool](../images/Lookup-tables1.png)
+
+### To view the contents of a lookup table: 
+Click on the eye icon for the lookup table you wish to view. 
+![Loopup tables - view](../images/Lookup-tables2.png)
+
+In each Look up table page, there are options to import and export, which is also how the table can be updated. More on [ Editing lookup tables]( #editing-lookup-tables).
+![Loopup tables - view](../images/Lookup-tables3.png)
+
+## Creating lookup tables
+Lookup tables are added to the system by importing csv files. 
+
+### To create a new lookup table: 
+1. Click on the ‘Add new lookup table’ button
+2. Select your csv file to import.
+![Loopup tables - csv file](../images/Lookup-tables4.png)
```

Comment:
If you want these to go within the list, you need to use proper Markdown line breaks (double space at end of the line)